procursus-deploy-linux-macos.sh

```diff
@@ -73,6 +73,7 @@ SNAPSHOT=$(snappy -s | cut -d ' ' -f 3 | tr -d '\n')
 
 snappy -f / -r "$SNAPSHOT" -t orig-fs > /dev/null 2>&1
 /prep_bootstrap.sh
+/usr/libexec/firmware
```

Comment:
`prep_bootstrap.sh` already runs `firmware`, what's the need for adding this line?